src/search/landmarks/landmark_cost_assignment.cc

```diff
@@ -174,10 +172,14 @@ lp::LinearProgram LandmarkEfficientOptimalSharedCostAssignment::build_initial_lp
 }
 
 double LandmarkEfficientOptimalSharedCostAssignment::cost_sharing_h_value(
```

Comment:
Have you tested this heuristic?

Reply:
Yes, https://ai.dmi.unibas.ch/_experiments/ai/downward/issue1036/data/issue1036-base-optimal-eval/issue1036-base-optimal-compare.html has a triple of `*-seq-opt-bjolp-opt` configurations which use this heuristic with CPLEX. Would you like to see some more tests?